### PR TITLE
Add OWNERS file and Kubeflow one-pager

### PR DESCRIPTION
**Pull request by texasmichelle**
Add OWNERS file.

PDF used as a handout at the NASA Frontier Development Lab event on Sept. 16, 2018.
Source located on the Kubeflow.org team drive:
  https://drive.google.com/open?id=1YLqH0e4dPyVCeivaz2QpBrhGNndbb_lD0uCPeaPr7pA

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/marketing-materials/3)

### CODE REVIEW

File: OWNERS [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# TODO(jlewi): We should probably have OWNERs files in subdirectories that 
+# list approvers for individual components (e.g. Seldon folks for Seldon component)
+approvers:
+  - aronchick
+  - jlewi
+  - texasmichelle
+reviewers:
+  - aronchick
+  - carmine
+  - jlewi
```
Comment (jlewi) [inline OWNERS]: Don't duplicate approvers in reviewers; (approvers are automatically reviewers). So remove aronchick, jlewi, and texasmichelle as reviewers

Comment (texasmichelle) [reply OWNERS]: Done

**Comment by texasmichelle** [comment]
@jlewi do you have write access to this repo? Can you merge directly?

**Pull request merged**